src/components/Modals/FiatRamps/views/Overview.tsx

```diff
@@ -170,26 +175,26 @@ export const Overview: React.FC<OverviewProps> = ({
             variant='outline'
             height='48px'
             justifyContent='space-between'
-            onClick={() => onIsSelectingAsset(selectedAsset, selectAssetTranslation)}
+            onClick={() => onIsSelectingAsset(assetId, selectAssetTranslation)}
```

Comment:
Might as well while diffing this line
```suggestion
            onClick={() => handleIsSelectingAsset(assetId, selectAssetTranslation)}
```